.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: 
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.version }}, ${{ matrix.gemfile }}
+    strategy:
+      matrix:
+        version: [2.4, 2.5, 2.6, 2.7]
+        gemfile: [Gemfile_ar41, Gemfile_ar50, Gemfile_ar51, Gemfile_ar_master]
+        exclude:
+          - version: 2.7
+            gemfile: Gemfile_ar41
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby ${{ matrix.version }}
+        uses: ruby/setup-ruby@v1
+        with: 
+          ruby-version: ${{ matrix.version }}
+      - name: Install dependencies
+        run: |
+          export BUNDLE_GEMFILE=${GITHUB_WORKSPACE}/${{ matrix.gemfile }}
+          rm -f ${GITHUB_WORKSPACE}/Gemfile.lock
```

Comment:
What error are you seeing when this isn't present?

I think it could be confusing if there are dependency-related failures and it's not clear the committed `Gemfile.lock` isn't being used.

Reply:
The workflow won't use the files: Gemfile_ar41, Gemfile_ar50, Gemfile_ar51, Gemfile_ar_master when this line isn't present :/ I'm not sure how else to go about it

Reply:
See https://github.com/Shopify/shopify_api/runs/1476242755#step:5:7 for the failure msg when `Gemfile.lock` isnt deleted. It for some reason defaults to activesupport-6.0.3.1?